<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_interface</class>
 <widget class="QDialog" name="Dialog_interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget,
QMainWindow{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(80, 80, 90, 255), stop:1 rgba(30, 30, 30, 255));
    font: 12pt &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
}
QTableWidget{
	alternate-background-color: rgba(255, 255, 255, 20);
}

/*表头样式*/

QHeaderView::section{
	background-color: rgb(86, 86, 86);
	border:1px solid rgba(220,220,220,100);
	color: rgb(255, 255, 255);

}
/*按钮*/
QPushButton,
QToolButton{
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(68, 68, 75, 255), stop:1 rgba(30, 30, 30, 255));
	border:1px groove rgba(255,255,255,100);

}
/*按钮悬停和按下状态*/
QPushButton:hover,
QPushButton:pressed,
QPushButton:checked,
QPushButton:selected,
QToolButton:hover,
QToolButton:pressed,
QToolButton:checked,
QToolButton:selected{
	color: rgba(255, 150, 0,255);

	background-color: qradialgradient(spread:pad, cx:0.505682, cy:0, radius:0.8, fx:0.505682, fy:0, stop:0 rgba(106, 162, 214, 255), stop:1 rgba(30, 30, 30, 255));
	border:2px groove rgba(70, 140, 195,100);
}
/*滚动条*/
QScrollBar{
	background-color: rgba(255, 255, 255,10);
	border:1px groove  gay;	
	border-color: rgba(255, 255, 255,80);
}
/*水平滚动条*/
QScrollBar:horizontal{
    height:17px;
    margin:0 16 0 16;
}
/*水平滚动条 滑块*/
QScrollBar::handle:horizontal{
	background-color: rgba(255,255,255,95);
    min-width:20px;
    margin:1px 0px 1px 0px;
	border:1px groove  gay;	
	border-color: rgba(255, 255, 255,80);
}
QScrollBar::handle:horizontal:hover{
	background-color:rgba(70, 140, 195,100);	
    min-width:20px;
    margin:1px 0px 1px 0px;
	border:1px groove  gay;	
	border-color: rgba(255, 255, 127,50);
}
/*水平滚动条 微调按钮*/
QScrollBar::add-line:horizontal{
	background-color: rgba(255,255,255,60);
    border:1px groove  gay;	
	border-color: rgba(255, 255, 255,80);
    width:15px;
    subcontrol-position:right;
    subcontrol-origin:margin;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0" columnstretch="3,1,9,0" columnminimumwidth="0,0,0,0">
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>备注信息：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_bz">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>功能号描述：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_ms">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1" colspan="3">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>入参：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget_rc">
       <column>
        <property name="text">
         <string>参数名称</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>参数描述</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>数据类型</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>备注信息</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>是否必须</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>操作</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1" colspan="3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>出参：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget_cc">
       <column>
        <property name="text">
         <string>参数名称</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>参数描述</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>数据类型</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>备注信息</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>操作</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label_top">
     <property name="styleSheet">
      <string notr="true">font: 75 18pt &quot;楷体&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>功能号：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
